pages/guides/getting-started/redwoodjs-guide.mdx

```diff
@@ -1,67 +1,77 @@
 tags: ['redwoodjs']
-author: estheragbaje
+author: estheragbaje, philzen
 ---
 
-### 1. Installation
+## Installation
 
-Inside your `Redwoodjs` project , cd into `web`, then, install Chakra UI by
-running either of the following:
+### Automatic
 
-```bash
-npm i @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^6
-```
+RedwoodJS provides a convenient setup script via its [CLI tool](https://redwoodjs.com/docs/cli-commands#setup-ui) 
+to enable Chakra UI in your project with a single command. In your project folder, simply enter:
 
 ```bash
-yarn add @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^6
+yarn redwood setup ui chakra-ui
 ```
 
-### 2. Provider Setup
+This will make the necessary modifications to your `App.tsx` (or `App.js`, respectively) 
+which are basically the same as outlined in manual installation steps below.
 
-After installing Chakra UI, you need to set up the `ChakraProvider` at the root
-of your application.
+See [additional setup](#additional-setup) (such as including 
+theme customizations and Storybook integration) below.
 
-Go to `App.js` and add the `ChakraProvider` like this:
+### Manual
 
-```jsx live=false
-import { ChakraProvider } from '@chakra-ui/react'
+1. Inside your `Redwoodjs` project , cd into `web`, then, install Chakra UI by
+running either of the following:
 
-const App = () => (
-  <ChakraProvider>
-    <FatalErrorBoundary page={FatalErrorPage}>
-      <RedwoodApolloProvider>
-        <Routes />
-      </RedwoodApolloProvider>
-    </FatalErrorBoundary>
-  </ChakraProvider>
-)
+   ```bash
+   npm i @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^6
+   ```
 
-export default MyApp
-```
+   ```bash
+   yarn add @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^6
+   ```
 
-### ChakraProvider Props
+2. After installing Chakra UI, you need to set up the `ChakraProvider` at the root
+of your application.  Go to `App.js` and add the `ChakraProvider` like this:
 
-| Name             | Type             | Default               | Description                                         |
-| ---------------- | ---------------- | --------------------- | --------------------------------------------------- |
-| resetCSS         | `boolean`        | `true`                | automatically includes `<CSSReset />`               |
-| theme            | `Theme`          | `@chakra-ui/theme`    | optional custom theme                               |
-| colorModeManager | `StorageManager` | `localStorageManager` | manager to persist a users color mode preference in |
-| portalZIndex     | `number`         | `undefined`           | common z-index to use for `Portal`                  |
+   ```jsx live=false
+   import { ChakraProvider } from '@chakra-ui/react'
+
+   const App = () => (
+     <FatalErrorBoundary page={FatalErrorPage}>
+       <RedwoodProvider titleTemplate="%PageTitle | %AppTitle">
+         <ColorModeScript />
```

Comment:
@Philzen you're using the `ColorModeScript` but you don't have it in the import statement at the top. I can notice that in the Customize Theme example as well.

Reply:
@nikolovlazar Fair point. However, the import is not required, i have a project up and running w/o an explicit import, and if i CTRL+click on `ColorModeScript` in VSCode, it resolves fine.

RedwoodJS does a lot of automagic import resolving under the hood, which alleviates the need for most imports you commonly would see in other projects.

https://redwoodjs.com/docs/tutorial/chapter2/routing-params states:
![grafik](https://user-images.githubusercontent.com/1634615/161801959-01efef81-ff6c-4ca6-a000-b23027d3821a.png)

That might be true for the Chakra UI dependencies as soon as they are added to a redwood project as well. Maybe @TimKolberger can shed some light on this?

Tim, would the import be required in any scenario (e.g. when installing it manually)?

Reply:
Good point, will fix this.

Reply:
OK, this is fixed now in e23beb2.

Good to have a final look at this, because i realised the export `MyApp` (which is also live on the website right now) is undefined :see_no_evil:  – subsequently fixed in c0e594f as well.